clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -282,7 +282,7 @@ export class HttpConnection implements IConnection {
     }
 
     private isITransport(transport: any): transport is ITransport {
-        return typeof (transport) === "object" && "connect" in transport;
+        return transport && typeof (transport) === "object" && "connect" in transport;
```

Comment:
This was needed because it fails if `transport` is `null`. That was true before my change too. `typeof null` is `object` (see below) but you can't use `in` with `null`, nor can you check for a method. There's a similar place we do this with `ILogger` that I'll fix as well.

![image](https://user-images.githubusercontent.com/7574/39490686-7b767ef6-4d3e-11e8-9d00-5ac7090d33a4.png)
![image](https://user-images.githubusercontent.com/7574/39490726-9a393464-4d3e-11e8-900b-035d90d2ea80.png)

![image](https://user-images.githubusercontent.com/7574/39490704-8661429c-4d3e-11e8-9ae2-00589363ca94.png)

Reply:
Put this comment in the code here https://github.com/aspnet/SignalR/pull/2179/files#diff-11502bbf017bbae00a723c7de894f310R350